### PR TITLE
Random Greetings for Players on the Play Page

### LINKED ISSUE #21: FEATURE: Changing Text on Play Page
## User Story
After this feature, I will:
- have changing text on my play page everytime i load it

## Screenshots
Talking about the "Welcome Farmer David Jr" section.
Screenshot 2023-08-11 at 1 30 53 AM

## Discussion
You know how in Minecraft, the subtext on the home menu changes every time you load into it? Yeah, that. That's the vision but for the text on the play page. Idk what we'd write, but I just think it'll be cool. We already greet the user in the commons page so why do it again here? Let's make it more fun!!

### PR DESCRIPTION
**Pull request by dscpsyl**
## Overview

In this PR, randomized greetings for the player are added on the play page to create additional interest in the game. Modeled after the home text screen of *Minecraft*, this allows for custom and routine messages to be displayed to the user every time they load the play page. Currently, there are 17 different lines that are included. The format `{welcome message} farmer {player name}!` has been chosen for simplicity.

## Screenshots (Optional)
*Note: Player name is not displayed as these are storybook screenshots*
Screenshot 2023-08-14 at 12 02 46 AM
Screenshot 2023-08-14 at 12 02 52 AM

## Feedback Request (Optional)
What do you guys think about the lines that are currently in the `PlayGreetings.json`? Feel free to add/suggest your own and give any feedback to the ones already included. This might not be a necessary or urgent feature, but I think it's a fun one that will add a little interest to the game.

## Validation (Optional)

1. Download this branch.
2. Run `npm run storybook`
3. Go to the play page.
4. Every time the page is loaded, a new line should appear.

## Tests

- [x] Frontend Unit tests (`npm test`) pass
- [x] Frontend Test coverage (`npm run coverage`) 100%
- [x] Frontend Mutation tests (`npx stryker run`) 100% 
- [x] Frontend Linting (`npx eslint --fix src`) 

## Linked Issues
Closes #21

### CODE REVIEW

**Review by yuanchenmeng — DISMISSED** [review]
I like this PR. Random greeting is much better than a fixed template.

File: frontend/src/main/components/Commons/CommonsPlay.js
Comment (chriscerie) [inline frontend/src/main/components/Commons/CommonsPlay.js]: Don't use an effect here. Calculate the random message in useState's initializer callback. Otherwise it will trigger an unnecessary render cycle and you get a flash of stale state.

Comment (dscpsyl) [reply frontend/src/main/components/Commons/CommonsPlay.js]: Thanks for the feedback! Fixed in [af19144](https://github.com/ucsb-cs156-m23/proj-happycows-m23-10am-4/pull/36/commits/af19144445c862664e6c41afd3e6e8083e77538c)

**Comment by publicqi** [comment]
@iain801 Can you take a look at the workflow?

**Review by publicqi — APPROVED** [review]
Code looks good to me.

**Comment by iain801** [comment]
merge from branch Iain-WorkflowFix to fix the error

**Comment by dscpsyl** [comment]
I will wait until that branch is merged and I'll rebase

**Review by jingpeng7527 — APPROVED** [review]
LGTM!

**Pull request merged**
